src/modules/user/controller/index.js

```diff
@@ -129,6 +129,23 @@ async function loginUser(req, res) {
   sendEmails(user.email);
   return res.status(201).send({ statusCode: 'CREATED', token, sendEmails });
 }
+/**
+ *
+ * @param {*} req ExpressRequest
+ * @param {*} res ExpressResponse
+ * @returns {*}  user || user not found errors
+ */
+async function logout(req, res) {
+  try {
+    // Remove the token from the cookies header
+    res.clearCookie('token');
+    // Send a response to the client
+    res.status(200).send({ message: 'Logged out successfully', token: null });
+  } catch (error) {
+    console.error(error);
```

Comment:
As you are returning the server error on response, you can remove this console.error(error); for easy manipulation.